### DISCUSSION TITLE
BTF-Lighting 5in1 LM052 Zigbee 3.0

### DISCUSSION THREAD

**Discussion by hesmith1029** [Koenkk/zigbee2mqtt#13546] [Devices]
Not sure where to report this info on sort of new device.  Bought a BTF-Lighting 5in1 LM052 Zigbee device on Amazon.  It claims to be Zigbee and also can be controlled with Mi-Light (Mi-Boxer)  controllers, specifically including a B8.   The box it comes in says it is a LAIMI Zigbee 3.0 5 in 1 LED Controller.   The device is working.

It paired up immediately with my Zigbee2mqtt installation in HomeAssistant.  I could control it just fine via HA.  It was discovered as Tuya TS0505B.

 Then shutdown my Zigbee controller and I was able to pair it to my B8 controller.   Was able to do all normal functions with the B8 controller.   The B8 operates independently of HA.   So if HA is down,  my wife can still control the LED strips from the B8.
 
 Plugged the Zigbee controller back in.   Could control the device from both HA and the B8.   However changes made by the B8 are not reported back over the Zigbee channel.   Maybe I just haven't found the correct Zigbee settings, who knows.    
 
Anyway, wanted to report this information.   Please let me know what other information you may need.    And if anyone has suggestions about getting the zigbee side to report back what has happen on the Mi-Light B8 side,  I would love to try them out.

**Comment by adgenet** [comment]
I had a couple of these LM052 controllers but encountered weirdness with regards to them not correctly changing color temperature and brightness consistently (setting color temperature would cause the controller to switch to max brightness without reporting the change back for example).
There turned out to be two versions of the tuya microcontroller modules in these.
The variant silkscreened "ZT3L" exhibited these issues. Others which use a "ZS3L" seem to function correctly.
Perhaps this is related to your issue as well.
I have since switched to different controllers because I wasn't able to find more ZS3L variants (I believe ZT3L versions have replaced the ZS3L versions) so I can't provide any additional input.

**Reply by Tommixoft** [reply]
if it's not secret what have you bought for replacement and now having no problem?
